libgsf.yaml

```diff
@@ -0,0 +1,89 @@
+package:
+  name: libgsf
+  version: 1.14.51
+  epoch: 0
+  description: Utility library for reading and writing structured file formats
+  copyright:
+    - license: LGPL-2.1-only
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - autoconf-archive
+      - automake
+      - build-base
+      - busybox
+      - bzip2-dev
+      - ca-certificates-bundle
+      - glib-dev
+      - gobject-introspection-dev~1.78
+      - gtk-doc
+      - intltool
+      - libtool
+      - libxml2-dev
+      - m4
+      - pkgconf
+      - pkgconf-dev
+      - python3
+      - zlib-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: mangled-package-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.gnome.org/GNOME/libgsf.git
+      expected-commit: 068a3a6da775c1a70fa43b2334b1b06afc2cf8ab
+      tag: LIBGSF_${{vars.mangled-package-version}}
+
+  - runs: |
+      sed -i -e 's/gsf-gnome//' -e 's/thumbnailer//' Makefile.*
+      NOCONFIGURE=1 ./autogen.sh
+
+  - runs: |
+      ./configure \
+      --build=$CBUILD \
+      --host=$CHOST \
+      --prefix=/usr \
+      --sysconfdir=/etc \
+      --localstatedir=/var \
+      --mandir=/usr/share/man \
+      --disable-static \
+      --without-gnome-vfs \
+      --without-bonobo \
+      --enable-introspection \
+      --enable-gtk-doc
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - runs: |
+      rm -f "${{targets.destdir}}"/usr/share/man/man1/gsf-office-thumbnailer.1
+
+  - uses: strip
+
+subpackages:
+  - name: libgsf-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - libgsf
+        - bzip2-dev
+    description: libgsf dev
+
+  - name: libgsf-doc
+    pipeline:
+      - uses: split/manpages
+    description: libgsf manpages
+
+# we use gitlab for this package, so we need to disable the release monitor otherwise CI fails
+update:
+  enabled: false
+  manual: true
```

Comment:
We should still be able to use auto updates even if the source is on gitlab.

Reply:
you can still configure the update section https://release-monitoring.org/project/1980/

and also include a nolint at the top of the file and the other's in this PR, like this https://github.com/wolfi-dev/os/blob/041b90b0027949e4ab82037d21181bdcb27858fe/gitlab-runner.yaml#L4C1-L4C45

Reply:
great learning! noted it!